index.js

```diff
@@ -0,0 +1,84 @@
+import addBook from './modules/addBook.js';
+import listBooks from './modules/listBooks.js';
+import removeBook from './modules/removeBook.js';
+import { DateTime } from './node_modules/luxon/src/luxon.js';
+
+const addButton = document.getElementById('addbtn');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const displayBook = document.getElementById('bookslist');
+const container = document.getElementById('container');
+const addbook = document.getElementById('addbook');
+const contact = document.getElementById('contact');
+const addnewbook = document.getElementById('addnewsec');
+const listbook = document.getElementById('listsec');
+const showcontact = document.getElementById('contactsec');
+const removeButtons = document.getElementsByClassName('btn-remove');
+
+function removeMyBook(i) {
```

Comment:
- [ ] As this is a practice for es6 model syntax let's kindly convert this to an arrow function.

Reply:
Fixed it.

Reply:
Thank you for your help as pointed out the specific lines